src/frontend/styles/components/banner.css

```diff
@@ -0,0 +1,40 @@
+
+.banner-container {
+    overflow-x: hidden;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 350px;
+    grid-column-gap: 0px;
+    grid-row-gap: 0px;
+    align-content: center;
+    justify-content: center;
+  }
+  .banner-image-div,
+  .banner-overlay-div,
+  .banner-text-div {
+    grid-area: 1 / 1 / 2 / 2;
+  }
+  
+  .banner-image {
+    display: grid;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+  }
+  
+  .banner-overlay-div {
+    display: grid;
+    max-width: 100%;
+    background: black;
+    background: linear-gradient(
+      60deg,
+      rgba(0, 0, 0, 0.7777485994397759) 30%,
+      rgba(255, 255, 255, 0) 100%
+    );
```

Comment:
Try not to use color names directly, use css variables instaed for consistency.

Reply:
thanks Om, will change this to a variable i, didn't did it initally as this is the one and only place its being used .